deployability/modules/jobflow/tests/test_workflow_file.py

```diff
@@ -6,17 +6,17 @@
 from unittest.mock import patch, MagicMock, call, mock_open
 import pytest
 
-from workflow_engine.workflow_processor import WorkflowFile
+from jobflow.workflow_processor import WorkflowFile
```

Comment:
Shouldn't the name of the workflow_processor method and WorkflowFile class name be changed?

Reply:
Modified in commit https://github.com/wazuh/wazuh-qa/pull/5460/commits/7910451c626e2a0b2a90e890e18ea93eb8b860a8